torchci/lib/bot/mergeBot.ts

```diff
@@ -89,7 +92,7 @@ function mergeBot(app: Probot): void {
       }
 
       if (ctx.payload.comment.user.login == ctx.payload.issue.user.login || await comment_author_in_pytorch_org()) {
-        await dispatchEvent("try-rebase");
+        await dispatchEvent("try-rebase", false, false, false, "", stable);
```

Comment:
Lol looking at this makes me wonder if we should be hosting this rebase code NOT in mergeBot.ts, but that's not a concern of this PR/task.